disable update check in all menus as well as update check on startup (see #626187)
TODO: in 1.3.x, the menus are stored in mindmaps => how to patch this?
--- a/freeplane/src/org/freeplane/main/application/FreeplaneGUIStarter.java
+++ b/freeplane/src/org/freeplane/main/application/FreeplaneGUIStarter.java
@@ -167,7 +167,7 @@
 			LinkController.install();
 			IconController.install();
 			HelpController.install();
-			controller.addAction(new UpdateCheckAction());
+			//controller.addAction(new UpdateCheckAction());
 			controller.addAction(new NextNodeAction(Direction.FORWARD));
 			controller.addAction(new NextNodeAction(Direction.BACK));
 			controller.addAction(new NextNodeAction(Direction.FORWARD_N_FOLD));
--- a/freeplane/resources/xml/browsemodemenu.mm
+++ b/freeplane/resources/xml/browsemodemenu.mm
@@ -483,6 +483,7 @@
 <attribute NAME="type" VALUE="submenu"/>
 <attribute NAME="name" VALUE="help"/>
 <attribute NAME="name_ref" VALUE="help"/>
+<!--
 <node TEXT="update" FOLDED="true" ID="ID_972374411" CREATED="1370343428377" MODIFIED="1370343428377">
 <attribute NAME="type" VALUE="category"/>
 <attribute NAME="name" VALUE="update"/>
@@ -491,6 +492,7 @@
 <attribute NAME="action" VALUE="UpdateCheckAction"/>
 </node>
 </node>
+-->
 <node TEXT="Web resources" FOLDED="true" ID="ID_902808063" CREATED="1370343428377" MODIFIED="1370343428377">
 <attribute NAME="type" VALUE="category"/>
 <attribute NAME="name" VALUE="Web resources"/>
--- a/freeplane/resources/xml/filemodemenu.mm
+++ b/freeplane/resources/xml/filemodemenu.mm
@@ -367,6 +367,7 @@
 <attribute NAME="type" VALUE="submenu"/>
 <attribute NAME="name" VALUE="help"/>
 <attribute NAME="name_ref" VALUE="help"/>
+<!--
 <node TEXT="update" FOLDED="true" ID="ID_1178888256" CREATED="1370196341118" MODIFIED="1370196341118">
 <attribute NAME="type" VALUE="category"/>
 <attribute NAME="name" VALUE="update"/>
@@ -375,6 +376,7 @@
 <attribute NAME="action" VALUE="UpdateCheckAction"/>
 </node>
 </node>
+-->
 <node TEXT="Web resources" FOLDED="true" ID="ID_911592986" CREATED="1370196341125" MODIFIED="1370196341125">
 <attribute NAME="type" VALUE="category"/>
 <attribute NAME="name" VALUE="Web resources"/>
--- a/freeplane/resources/xml/mindmapmodemenu.mm
+++ b/freeplane/resources/xml/mindmapmodemenu.mm
@@ -1592,6 +1592,7 @@
 <attribute NAME="type" VALUE="submenu"/>
 <attribute NAME="name" VALUE="help"/>
 <attribute NAME="name_ref" VALUE="help"/>
+<!--
 <node TEXT="update" FOLDED="true" ID="ID_722782407" CREATED="1370194798905" MODIFIED="1370194798905">
 <attribute NAME="type" VALUE="category"/>
 <attribute NAME="name" VALUE="update"/>
@@ -1600,6 +1601,7 @@
 <attribute NAME="action" VALUE="UpdateCheckAction"/>
 </node>
 </node>
+-->
 <node TEXT="---" ID="ID_1292710565" CREATED="1370194798906" MODIFIED="1370194798906">
 <attribute NAME="type" VALUE="separator"/>
 </node>
